packages/postcss-normalize-positions/src/index.js

```diff
@@ -156,25 +156,27 @@ function transform(value) {
       return;
     }
 
-    if (firstNode === 'center' && directionKeywords.has(secondNode)) {
-      nodes[0].value = nodes[1].value = '';
+    if (secondNode !== null) {
```

Comment:
`secondNode` gets explicitly assigned to `null` in some cases before, so all the following code needs this check.